Maths/CollatzSequence.js

```diff
@@ -26,5 +26,5 @@ export function collatz (n) {
     steps.push(n)
   }
 
-  return { result: n, steps: steps }
+  return { result: n, steps }
```

Comment:
Unrelated change, move this to a separate PR please.

Reply:
Sorry new to git. Not sure why have changes on the Maths/CollatzSequence.js...